lighthouse-core/fraggle-rock/gather/timespan-runner.js

```diff
@@ -63,6 +63,7 @@ async function startTimespan(options) {
           url: finalUrl,
           config,
           computedCache,
+          gatherMode: 'timespan',
```

Comment:
for clarification, possibly for future refactors: it does seem kind of weird that this is in the (optional) `FRContext` and used to create the config, but is manually specified here, `phaseOptions`, and as back up if an `FRContext` isn't provided. Should it just be pulled off the config after config init? Why is it an option at all?

Reply:
https://github.com/GoogleChrome/lighthouse/pull/12850#discussion_r688708879